cased-rails.gemspec

```diff
@@ -19,12 +19,12 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'cased-ruby', '~> 0.8.0'
```

Comment:
Originally this line was removed but we actually need it in order to add cased-ruby